articles/foreign-host-vitals-identity-provider-idp-user-information.md

```diff
@@ -0,0 +1,283 @@
+# Foreign host vitals: identity provider (IdP) user information
+
+![Import users from IdP to Fleet](../website/assets/images/articles/add-users-from-idp-cover-img.png)
+
+_Available in Fleet Premium._
+
+Fleet supports [Okta](#okta), [Microsoft Active Directory (AD) / Entra ID](#microsoft-entra-id), [Google Workspace](#google-workspace), [authentik](#google-workspace), as well as any other identity provider (IdP) that supports the [SCIM](https://scim.cloud/) standard.
+
+To add IdP host vitals, follow steps for your IdP.
```

Comment:
```suggestion
To add IdP host vitals, like the end user's groups and full name, follow steps for your IdP. Fleet supports [Okta](#okta), [Microsoft Active Directory (AD) / Entra ID](#microsoft-entra-id), [Google Workspace](#google-workspace), [authentik](#google-workspace), as well as any other identity provider (IdP) that supports the [SCIM](https://scim.cloud/) standard.
```

After reading this again, I think we swap sentences to start with what the user wants to do. Add end user info.